### PR TITLE
Update README + Docker Compose Bundle references

### PR DESCRIPTION
**Pull request by cdrage**
This commit updates a few sentences in the README that I thought would
need improving.

Bundles now end with the .dab extension and are named "Distributed
Application Bundles" now. I've updated the README as well as the example
accordingly.

### CODE REVIEW

**Comment by surajssd** [comment]
That is awesome clean up @cdrage Thanks for it :+1:

**Comment by kadel** [comment]
LGTM :+1: . But I definitely shouldn't be the one reviewing anything written English :smiley:

**Comment by ngtuna** [comment]
That's also lgtm. You might want to take a look @sebgoa

File: README.md [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
 ```console
-$ kompose --bundle docker-compose-bundle.dsb convert
+$ kompose --bundle docker-compose-bundle.dab convert
+WARN[0000]: Unsupported key networks - ignoring
+file "redis-svc.json" created
+file "web-svc.json" created
+file "web-deployment.json" created
+file "redis-deployment.json" created
+
+$ kompose docker-compose-bundle.yml convert
```
Comment (sebgoa) [inline README.md]: is that a typo ? shouldn't it be docker-compose.yml ?

File: README.md [hunk 2]
```diff
@@ -1,29 +1,43 @@
-# Kubernetes compose (Kompose)
+# Kompose (Kubernetes + Compose)
 
 [![Build Status](https://travis-ci.org/skippbox/kompose.svg?branch=master)](https://travis-ci.org/skippbox/kompose) [![Join us on Slack](https://s3.eu-central-1.amazonaws.com/ngtuna/join-us-on-slack.png)](https://skippbox.herokuapp.com)
 
-`kompose` is a tool to help users familiar with `docker-compose` move to [Kubernetes](http://kubernetes.io). It takes a Docker Compose file and translates it into Kubernetes resources.
+`kompose` is a tool to help users whom are familiar with `docker-compose` move to [Kubernetes](http://kubernetes.io). `kompose` takes a Docker Compose file and translates it into Kubernetes resources.
```
Comment (sebgoa) [inline README.md]: I will assume you are a native english speaker, and that the use of `whom` here is correct.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
 
-`kompose` is a convenience tool to go from local Docker development to managing your application with Kubernetes. We don't assume that the transformation from docker compose format to Kubernetes API objects will be perfect, but it helps tremendously to start _Kubernetizing_ your application.
+`kompose` is a convenience tool to go from local Docker development to managing your application with Kubernetes. Transformation of the Docker Compose format to Kubernetes API objects may not be 100% perfect, but it helps tremendously when first deploying to Kubernetes.
```
Comment (cdrage) [inline README.md]: yeah, that's good! i'll update it to that

File: README.md (hunk 2 above)
Comment (cdrage) [inline README.md]: yup :)

**Comment by cdrage** [comment]
@sebgoa updated it with your changes!

**Pull request merged**